include/librealsense2/hpp/rs_export.hpp

```diff
@@ -81,76 +111,174 @@ namespace rs2
             auto width = profile.width(), height = profile.height();
             static const auto threshold = 0.05f;
```

Comment:
Is it possible to add the threshold to the list of PLY options ? This will allow for better fine-tune of resulted models